mysqld_exporter.go

```diff
@@ -175,14 +186,19 @@ func (e *Exporter) setMetrics(scrapes <-chan []string) {
 			continue
 		}
 
-		if _, ok := e.metrics[name]; !ok {
-			e.metrics[name] = prometheus.NewGauge(prometheus.GaugeOpts{
-				Namespace: namespace,
-				Name:      name,
-			})
-		}
+		command := comRegex.FindStringSubmatch(name)
+		if len(command) > 0 {
+			e.commands.With(prometheus.Labels{"command": command[1]}).Set(float64(value))
```

Comment:
Ok... They were all separate metrics before, and now they're the same number of time series, just grouped into one metric (which seems to makes sense). I think that should be ok, and since @brian-brazil hasn't objected, :+1: :)